.github/workflows/comment_bot.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           git remote add upstream https://github.com/apache/arrow
           git fetch upstream
           changed() {
-            git diff --name-only HEAD^..upstream/master | grep -e "$1" >/dev/null 2>&1
+            git diff --name-only HEAD..upstream/master | grep -e "$1" >/dev/null 2>&1
```

Comment:
I did test this but only with a single commit and I did at first see no changes until I stumbled upon the `^` here. @nealrichardson do you remember why you did omit the one commit?

Reply:
I think I copy-pasted from somewhere; git syntax is not my strength. I've known it's not quite right for a while but not sure how to fix it, so I'm assuming your change is more correct.